packages/standard-components/src/forms/InnerForm.svelte

```diff
@@ -108,8 +116,28 @@
 
       return !newError
     }
+
+    const clearValue = () => {
+      const { fieldState } = fieldMap[field]
+      fieldState.update(state => {
+        state.value = defaultValue
```

Comment:
Using the new variable declared above, we can update this to:
`state.value = newValue`
to accound for initial values and default values.